### ISSUE TITLE
Setting the volume on an `AudioInstance` instead of `PlayAudioCommand` doesn't set it relative to a channel.

### ISSUE THREAD

**Issue by Selene-Amanita** [NiklasEi/bevy_kira_audio#110]
Follow up of https://github.com/NiklasEi/bevy_kira_audio/issues/103 / https://github.com/NiklasEi/bevy_kira_audio/pull/106

If you have an `AudioChannel` called `my_audio_channel` with volume `0.5` let's say.

The fix works if you do `my_audio_channel.play(my_handle_audio_source).with_volume(0.2);`, it would set the volume to `0.1`, as expected.

But if you do:
```rust
    let my_handle_audio_instance = my_audio_channel.play(my_handle_audio_source).looped().paused().handle()
    ///...

/// Then in another system
fn other_system (
    mut audio_instances: ResMut<Assets<AudioInstance>>,
    //...
) {
    //... get start_sound_handle somehow
    let sound = audio_instances.get_mut(start_sound_handle).unwrap();
    sound.resume(AudioTween::default());
    sound.set_volume(0.2, AudioTween::default());
}```
then the volume will be set to 0.2 instead, I think, I do basically that and when the volume of the channel is `0.` I still hear the sound.

**Comment by NiklasEi** [comment]
Yes you are right, this was an oversight on my end in #106. Thanks for the issue, I will adapt the behavior for setting the volume on an instance.

**Comment by PPakalns** [comment]
Have the same problem. In my case I have implemented spatial audio manually and need to manually update volume and panning. 

It is confusing that there are APIS where you need and don't need to manually take into channel volume.

**Comment by nixon-voxell** [comment]
May I ask if this issue is resolved and how should it be handled now? (I'm using spatial audio as well)

**Comment by PikminGuts92** [comment]
Thank you for the quick response! I was able to use the workaround as you said. This is sufficient for my purposes.
